Code/Tools/FBuild/FBuildCore/Graph/NodeGraph.cpp

```diff
@@ -1466,7 +1466,6 @@ bool NodeGraph::CheckDependencies( Node * nodeToBuild, const Dependencies & depe
 
     // sanity checks
     ASSERT( cleanPath.Find( OTHER_SLASH ) == nullptr ); // bad slashes removed
-    ASSERT( cleanPath.Find( NATIVE_DOUBLE_SLASH ) == nullptr ); // redundant slashes removed
```

Comment:
For clarification: now that I am trying to support UNC paths, double slashes can (and will) appear in correctly cleaned paths.